src/FlexTable.jl

```diff
@@ -243,24 +243,25 @@ end
 # Private fields are never exposed since they can conflict with column names
 Base.propertynames(t::FlexTable, private::Bool=false) = columnnames(t)
 
-function Base.vcat(t::Union{FlexTable, Table}, t2::Union{FlexTable, Table})
-    return FlexTable{_vcat_ndims(ndims(t), ndims(t2))}(map(vcat, columns(t), columns(t2)))
+
+@inline function Base.vcat(ts::Union{FlexTable, Table}...)
```

Comment:
Did you check the *specificity* here with respect to things like `vcat(::AbstractVector, ::AbstractVetor)`? I can't remember if arity or namy-ness™ takes precedence, or if it's ambiguous. I think `@which` can help identify what actually get's called.

I suppose the tests should catch ambiguities in theory. There's also a tool called `detect_ambiguities`. Try `using Test, TypedTables; detect_ambiguities(Base, TypedTables)` - this can help a bit in identifying potential issues. I believe there are some existing ones I need to fix but there shouldn't be anything for `vcat` or `hcat`.

Reply:
Looks fine, for all tests cases our methods are called, not the Base defaults.

`detect_ambiguities` reports multiple ambiguities for `checkbounds` and one for `map`, nothing else.

Reply:
Perfect - thanks for checking.